sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,121 @@
     </item>
     <item>
-      <EFDB_Name>extraVisit</EFDB_Name>
-      <Description>Extra-focal image visit number.</Description>
-      <IDL_Type>int</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>intraDirectoryPath</EFDB_Name>
-      <Description>Absolute directory path to intra-focal wavefront data collection.</Description>
+      <EFDB_Name>config</EFDB_Name>
+      <Description>Yaml configuration string with additional parameters to be passed to ISR and other pre-process tasks. If options are invalid command will be rejected. If not given, use the standard configuration.</Description>
       <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_command_runWEP</EFDB_Topic>
+    <Description>Process wavefront data.</Description>
     <item>
-      <EFDB_Name>extraDirectoryPath</EFDB_Name>
-      <Description>Absolute directory path to extra-focal wavefront data collection.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
+      <EFDB_Name>visitId</EFDB_Name>
+      <Description>Image visit id number. This is a unique identifier for each image and consists of the date of the observation plus an integer iterator, e.g. 2020103000040 for the 40th image taken in October, 30, 2020.</Description>
+      <IDL_Type>long</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>fieldRA</EFDB_Name>
-      <Description>Right ascension of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>fieldDEC</EFDB_Name>
-      <Description>Declination of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>filter</EFDB_Name>
-      <Description>Active filter type. Use 1-6 for u, g, r, i, z, and y. Use 7 for reference.</Description>
-      <IDL_Type>int</IDL_Type>
+      <EFDB_Name>extraId</EFDB_Name>
+      <Description>Extra focal image visit id number (optional). Same as visitId, if different than zero, the command assumes this is an Intra/Focal sequence, otherwise it assumes Corner Wavefront Sensor.</Description>
+      <IDL_Type>long</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
-    <item>
-      <EFDB_Name>cameraRotation</EFDB_Name>
-      <Description>Camera rotation angle.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
     <item>
       <EFDB_Name>userGain</EFDB_Name>
       <Description>Gain value between 0 and 1. Set to -1 to ignore user gain.</Description>
-      <IDL_Type>double</IDL_Type>
+      <IDL_Type>float</IDL_Type>
```

Comment:
It is the `runOFC()` needs the `userGain`, not the `runWEP()`.

Reply:
good catch.